app/assets/stylesheets/darkswarm/hub_node.css.scss

```diff
@@ -8,6 +8,14 @@
       white-space: nowrap;
       overflow-x: hidden;
       overflow-y: visible;
+
+      i {
+        margin-right: 0.5rem;
+      }
+    }
+
+    .active_table_row {
```

Comment:
I was wondering if this doesn't allow the rows to grow in case they need two lines of text, but the current design doesn't allow for that either, it just cuts the shop name. So this isn't doing any harm, I think. This may be something to address somewhere else.

Before:
![screenshot from 2018-01-24 15-44-04](https://user-images.githubusercontent.com/3524483/35314928-828a9276-011d-11e8-95b4-16b44b75e9ce.png)

After (different data):
![screenshot from 2018-01-24 15-59-41](https://user-images.githubusercontent.com/3524483/35315282-b03cc9d0-011f-11e8-8e79-0774849ac4fc.png)